### ISSUE TITLE
audit for backticked package names

### ISSUE THREAD

**Issue by simonpcouch** [tidymodels/parsnip#1134]
In many places in parsnip, the package name is written "`parsnip`" rather than "parsnip" (i.e. there are backticks before and after). Per [tidy style](https://style.tidyverse.org/index.html), package names don't need backticks.

Here's an example GitHub search:

https://github.com/search?q=repo%3Atidymodels%2Fparsnip+%22%60parsnip%60%22&type=code

Let's search for tidymodels and tidyverse package names and make sure they're not backticked.

The part that makes this a little more tricky than a Find and Replace is knowing when the thing should actually be backticked vs. not. If the text is definitely referencing parsnip _as a package_, take the backticks away. If there's e.g. a reference to the `tune()` function rather than the package tune, leave it as is (making sure to include parentheses following function names if needed). Be sure to `devtools::document()` and/or `devtools::build_readme()` before putting in the PR!

**Comment by topepo** [comment]
Some code that will help you run `document()`: 

```r
truly_req_pkgs <- c("glmnet", "Cubist", "xrf", "ape", parsnip:::extensions())

pak::pak(truly_req_pkgs, ask = FALSE)

# Before devtools::document()
inst <- purrr::map(rmd_pkgs, ~ library(.x, character.only = TRUE))
```

**Comment by luisDVA** [comment]
hi max @topepo ,
Currently in the process of running the documentation for my PR on this. The snippet above helped me install ~140 pkgs but some are still missing when I run `document()` so I'm installing them manually.  In the code above what was `rmd_pkgs` meant to refer to?

**Comment by simonpcouch** [comment]
Addressed in #1158. :)
